frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -45,6 +55,18 @@ export function InsightTimeoutState({
                     {isLoading ? <Animation type={AnimationType.SportsHog} /> : <IconErrorOutline />}
                 </div>
                 <h2>{isLoading ? 'Looks like things are a little slow…' : 'Your query took too long to complete'}</h2>
+                {isLoading && samplingAvailable && suggestedSamplingPercentage ? (
```

Comment:
Should this also be in the list of things you can do right below it? 🤔 

"Click the button above to sample" feels a bit weird to list it, but also weird to not have it in the list of things

Reply:
Yeah this is one bit I'm not super happy with. Maybe we can shorten the list (some things are no longer relevant) and make sure it's a prominent enough link? @lharries thoughts?

Reply:
Added some images below